### DISCUSSION TITLE
Bugged Score?

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#20487] [Q&A] [answered]
Beatmap in question: https://osu.ppy.sh/beatmapsets/1788950#mania/3665476
1.000.000 score with 93.45% acc and 7 misses while apparently no mods were active and combo 470/**0**.
Screenshot:
![image](https://user-images.githubusercontent.com/31819497/192247284-e2655143-9130-437a-b4c0-92e5e3d76c85.png)

**Comment by peppy** [comment]
Please attach your replay.

**Reply by SevenChords** [reply]
it wasn't my score and (assuming cause they aren't implemented yet) i can't download the replay of the score in question

**Comment by cdwcgt** [comment, marked as answer]
This man forged his score return to obtain theoretical value😅
but he forget to adjust rank.

osu-web do not check the legality of score

**Reply by cdwcgt** [reply]
ok nice he just change his score and leave all other value.

**Reply by smoogipoo** [reply]
Scores were all recalculated recently so this has been fixed.
